sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_constants.py

```diff
@@ -102,7 +102,9 @@ def _env_var_or_default(var_name, default_val=""):
 # Autoinstrumentation
 
 def _is_attach_enabled():
```

Comment:
Should this be a generic "are we in attach" function or only one that is specific to app service?

Reply:
Yes. But that is the only attach scenario we currently have. Given our conversations with functions, it seems functions codeless will look different and likely not have the /agents/ folder. So I decided to structure this util function for different cases.

Reply:
I can add a todo here if that would make that clearer

Reply:
I believe we should have a generic "is attach enabled" and then have branching logic depending on which environment you are in.

Reply:
Yeah that's exactly what this is. It just only has one branch.